tests/test_numpy_array.py

```diff
@@ -158,3 +159,53 @@ def test_make_c_f_array():
     assert not make_c_array().flags.f_contiguous
     assert make_f_array().flags.f_contiguous
     assert not make_f_array().flags.c_contiguous
+
+
+@pytest.requires_numpy
```

Comment:
Maybe a separate test for base lifetime? E.g., create base, bind array to it, del/decref base, make sure the array and base are still alive, kill array, make sure the base is also dead?